src/utilities.js

```diff
@@ -32,8 +33,73 @@ export const getIcon = incident => {
       return <Power className='element' />
     case 'hazmat spill':
       return <Hazmat className='hazard' />
+    case 'hurricane':
+      return <Hurricane className='hurricane' />
     default:
       break
   }
 }
 
+export const statesCodes = (type, setValue, value) => {
+  const name = `${type}-state`
```

Comment:
This was a good refactor - saves us many lines of code and keeps the IncidentForm and NewContactForm components trimmed down